.ci/enable_sandbox_cgroup_only.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o errtrace
+
+readonly script_dir=$(dirname $(readlink -f "$0"))
+# Source to trap error line number
+# shellcheck source=../lib/common.bash
+source "${script_dir}/../lib/common.bash"
+# shellcheck source=./lib.sh
+source "${script_dir}/lib.sh"
+
+current_value=$(kata-runtime kata-env --json | jq ".Runtime.SandboxCgroupOnly")
+if [ "$current_value" == "true" ]; then
+	echo "already enabled"
+	exit
+fi
+
+kata_config_path=$(kata-runtime kata-env --json | jq -r .Runtime.Config.Path)
+
+option="sandbox_cgroup_only"
+option_false="${option}=false"
+option_true="${option}=true"
+bk_suffix="${option}-bk"
+
+echo "config file base: ${kata_config_path}"
+
+if [ -f "${KATA_ETC_CONFIG_PATH}" ]; then
+	bk_file="${KATA_ETC_CONFIG_PATH}-${bk_suffix}"
+	echo "backup ${KATA_ETC_CONFIG_PATH} in ${bk_file}"
+	sudo cp "${KATA_ETC_CONFIG_PATH}" "${bk_file}"
+fi
+
+if [ "${KATA_ETC_CONFIG_PATH}" != "${kata_config_path}" ]; then
+	echo "Creating etc config based on ${kata_config_path}"
+	sudo cp "${kata_config_path}" "${KATA_ETC_CONFIG_PATH}"
+fi
+
+echo "Replacing ${option_false} for ${option_true}"
+# If is false
+sudo sed -i "s,^${option_false},${option_true},g" "${KATA_ETC_CONFIG_PATH}"
+# If is true but commented
+echo "Replacing #${option_true} for ${option_true}"
+# If is false and commented
+echo "Replacing #${option_false} for ${option_true}"
+sudo sed -i "s,^#${option_false},${option_true},g" "${KATA_ETC_CONFIG_PATH}"
+
+echo "Validate option is enabled : ${option_true}"
+current_value=$(kata-runtime kata-env --json | jq ".Runtime.SandboxCgroupOnly")
+if [ "$current_value" == "true" ]; then
+	echo "enabled"
+	exit
```

Comment:
Nit: I think it's always safest to specify the exit value to avoid the shell deciding for you (based on the result of the last command executed).